overlord/snapstate/handlers_prereq_test.go

```diff
@@ -136,16 +136,65 @@ func (s *prereqSuite) TestDoPrereqWithBaseNone(c *C) {
 	// check that the do-prereq task added all needed prereqs
 	expectedLinkedSnaps := []string{"prereq1", "snapd"}
 	linkedSnaps := make([]string, 0, len(expectedLinkedSnaps))
+	lane := 0
 	for _, t := range chg.Tasks() {
 		if t.Kind() == "link-snap" {
 			snapsup, err := snapstate.TaskSnapSetup(t)
 			c.Assert(err, IsNil)
 			linkedSnaps = append(linkedSnaps, snapsup.InstanceName())
+		} else if t.Kind() == "prerequisites" {
+			c.Assert(t.Lanes(), DeepEquals, []int{lane})
+			lane++
 		}
 	}
 	c.Check(linkedSnaps, DeepEquals, expectedLinkedSnaps)
```

Comment:
Maybe add a final check for expected `lane` value?

Reply:
Added